setup.py

```diff
@@ -60,35 +59,6 @@
     'cherrypy.scaffold',
     'cherrypy.wsgiserver',
 ]
-data_files = [
-    ('cherrypy', [
-        'cherrypy/cherryd',
-        'cherrypy/favicon.ico',
```

Comment:
I've reproduced your steps and found out that it throws `404 Not Found`:
```
$ curl -I http://localhost:8080/favicon.ico
HTTP/1.1 404 Not Found
Connection: close
Date: Tue, 27 Dec 2016 15:48:26 GMT
Server: 127.0.0.1
```
It looks like module you are running isn't using `quickstart` (I didn't really check.

It's better to check the tutorial's helloworld:
```
$ cd ~/src/cherrypy/cherrypy/tutorial/
$ python tut01_helloworld.py
[27/Dec/2016:17:52:13] ENGINE Listening for SIGUSR1.
[27/Dec/2016:17:52:13] ENGINE Listening for SIGTERM.
[27/Dec/2016:17:52:13] ENGINE Listening for SIGHUP.
[27/Dec/2016:17:52:13] ENGINE Bus STARTING
[27/Dec/2016:17:52:13] ENGINE Started monitor thread 'Autoreloader'.
[27/Dec/2016:17:52:13] ENGINE Started monitor thread '_TimeoutMonitor'.
[27/Dec/2016:17:52:13] ENGINE Serving on http://127.0.0.1:8080
[27/Dec/2016:17:52:13] ENGINE Bus STARTED
^Z
[1]  + 6003 suspended  python tut01_helloworld.py
$ bg
[1]  + 6003 continued  python tut01_helloworld.py
$ curl -I http://localhost:8080/favicon.ico
127.0.0.1 - - [27/Dec/2016:17:52:22] "HEAD /favicon.ico HTTP/1.1" 200 1406 "" "curl/7.46.0"
HTTP/1.1 200 OK
Content-Type: image/x-icon
Content-Length: 1406
Accept-Ranges: bytes
Date: Tue, 27 Dec 2016 15:52:22 GMT
Server: CherryPy/8.3.0.post20161225
Last-Modified: Mon, 25 Jul 2016 19:26:48 GMT
```